game-core/src/main/java/games/strategy/engine/chat/ChatMessageListener.java

```diff
@@ -4,13 +4,13 @@
 
 /** Callback interface for a component that is interested in chat messages. */
 public interface ChatMessageListener {
-  void addMessage(String message, PlayerName from);
+  void messageReceived(String message, PlayerName from);
 
-  void addSlapMessage(String message, PlayerName from);
+  void slap(String message, PlayerName from);
```

Comment:
slap and slap could be a bit confusing. One is sending a slap, the other is receiving a slap. I'd suggest "slap" and "slapped", the latter to indicate a slap has been received.

If you agree, please fix and then merge!  :shipit: